.dockstore.yml

```diff
@@ -184,6 +184,14 @@ workflows:
        branches:
          - master
          - ah_var_store
+   - name: GvsVariantCalling
+     subclass: WDL
+     primaryDescriptorPath: /scripts/variantstore/wdl/GvsUnified.wdl
```

Comment:
I think this was meant to be `scripts/variantstore/wdl/GvsJointVariantCalling.wdl`. Also should the name here be `GvsJointVariantCalling` or the WDL renamed to match this?